app/models/vm_retire_request.rb

```diff
@@ -1,7 +1,7 @@
 class VmRetireRequest < MiqRetireRequest
   TASK_DESCRIPTION  = 'VM Retire'.freeze
   SOURCE_CLASS_NAME = 'Vm'.freeze
-  ACTIVE_STATES     = %w(retired) + base_class::ACTIVE_STATES
+  ACTIVE_STATES     = %w(retired provisioned) + base_class::ACTIVE_STATES
```

Comment:
Why would a VmRetireRequest have a state of "provisioned"?

Reply:
Because naming is difficult and it's not the state of the request, it's the state of the object itself. I mean you may as well be asking why a VmRetireRequest can be retired, since the request itself doesn't work like that either. ¯\_(ツ)_/¯

Reply:
I mean, it was a good try 🤣